src/components/graphqlFormGenerator/FormGenerator.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <v-form validate>
+    <h3>{{ mutation.name }}</h3>
+    <p>{{ mutation.description }}</p>
+    <form-input
+     v-for="input in inputs"
+     v-bind:key="input.label"
+     v-model="model[input.label]"
+     :gqlType="input.gqlType"
+     :label="input.label"
+    />
+    <v-btn
+      @click="reset"
+    >
+      Reset
+    </v-btn>
+    <pre ref="output">{{ model }}</pre>
+  </v-form>
+</template>
+
+<script>
+import FormInput from '@/components/graphqlFormGenerator/FormInput'
+import cloneDeep from 'lodash/cloneDeep'
+
+export default {
+  name: 'form-generator',
```

Comment:
You've created a new object here and exported it - Can we have an over-arching docstring, or is that "just not very vue-js"?

Reply:
`export default` is a JavaScript pattern, export something in one file, import it in another.

The MDN pages are pretty good reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
I get that (I've been doing the codeacademy intro to JS and a course on vue) - I just wanted a docstring.